core/src/main/java/org/bitcoinj/evolution/AbstractQuorumState.java

```diff
@@ -90,7 +90,7 @@
 
     QuorumUpdateRequest<Request> lastRequest;
 
-    static final long WAIT_GETMNLISTDIFF = 5000;
+    static final long WAIT_GETMNLISTDIFF = 5;
```

Comment:
changes made previously are keeping track of the time of sent messages is now using seconds instead of milliseconds.